### PR TITLE
frontend: convert `SkipForTesting` to functional

### PR DESCRIPTION
**Pull request by shonsirsha**
Converting SkipForTesting to functional component.

### CODE REVIEW

File: frontends/web/src/routes/device/components/skipfortesting.tsx [hunk 1]
```diff
@@ -14,51 +14,40 @@
...
+import React, { useState } from 'react';
+import { testRegister } from '../../../api/account';
 import { Button } from '../../../components/forms';
 import { PasswordSingleInput } from '../../../components/password';
-import { apiPost } from '../../../utils/request';
 
-interface SkipForTestingProps {
+type TProps = {
     show: boolean;
 }
```
Comment (shonsirsha) [inline frontends/web/src/routes/device/components/skipfortesting.tsx:24]: I wonder if we need this at all? 🤔

Comment (thisconnect) [reply frontends/web/src/routes/device/components/skipfortesting.tsx:24]: good catch. this seems to be a left over from when it used show prop https://github.com/digitalbitbox/bitbox-wallet-app/blob/71a45aa2c6566bc1a50433a56750e7ee4f9778e2/frontends/web/src/routes/device/waiting.tsx#L60

File: frontends/web/src/api/account.ts [hunk 2]
```diff
@@ -317,4 +317,8 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
     coinCode,
     name,
   });
+};
+
+export const testRegister = (pin: string): Promise<any> => {
```
Comment (thisconnect) [inline frontends/web/src/api/account.ts:322]: It doesn't matter, but return type should be `Promise<null>`.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/88d28497cb56f05d63e6ea465201da1934a6fc38/backend/handlers/handlers.go#L681-L692

**Review by thisconnect — APPROVED** [review]
One more suggestion (return type change), then it LGTM

File: frontends/web/src/api/account.ts [hunk 3]
```diff
@@ -317,4 +317,8 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
     coinCode,
     name,
   });
+};
+
+export const testRegister = (pin: string): Promise<null> => {
```
Comment (benma) [inline frontends/web/src/api/account.ts:322]: nit: this belongs into the backend.ts not account.ts. account.ts is for account endpoints, which all have a account `code` param. same for `addAccount` above actually.

**Pull request merged**